README.md

```diff
@@ -1,5 +1,7 @@
 # xVimscript
 
+[![Join the chat at https://gitter.im/exercism/xvimscript](https://badges.gitter.im/exercism/xvimscript.svg)](https://gitter.im/exercism/xvimscript?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
```

Comment:
Could we change this to the TravisCI badge coming first and no blank line between both badges?  :pray:

That's how most projects handle it: CI badges first and all badges side by side.